<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50,50,50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stacked_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>351</width>
      <height>420</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="stacked_widget_tetris">
     <widget class="GridWidget" name="widget_tetris" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>210</width>
        <height>420</height>
       </rect>
      </property>
     </widget>
     <widget class="NextPlayerGridWidget" name="widget_tetris_next_player" native="true">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>0</y>
        <width>84</width>
        <height>84</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>84</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>84</width>
        <height>84</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_score_tetris">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_level_tetris">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Level:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Records:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_1">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>1.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>2.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>3.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>4.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>5.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_score_tetris_value">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_level_tetris_value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Level:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_1_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>1.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_2_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>2.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_3_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>3.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_4_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>4.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_5_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>5.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_state_tetris">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>400</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stacked_widget_snake">
     <widget class="GridWidget" name="widget_snake" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>210</width>
        <height>420</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_4_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>4.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>4.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_score_snake">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_score_snake_value">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_2_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>2.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>3.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_level_snake">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Level:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>5.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_5_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>5.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_level_snake_value">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Level:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_state_snake">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>400</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_3_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>3.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_1">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>1.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>Records:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_1_value">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>1.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_records_snake_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt</string>
      </property>
      <property name="text">
       <string>2.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stacked_widget_car_racing">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>car_racing</string>
      </property>
     </widget>
     <widget class="GridWidget" name="widget_car_racing" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>210</width>
        <height>420</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stacked_widget_tank_classic">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tank classic</string>
      </property>
     </widget>
     <widget class="GridWidget" name="widget_tank_classic" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>210</width>
        <height>420</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="push_button_tetris">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Tetris</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_snake">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Snake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_car_racing">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Car Racing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_tank_classic">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Tank Classic</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GridWidget</class>
   <extends>QWidget</extends>
   <header>QWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NextPlayerGridWidget</class>
   <extends>QWidget</extends>
   <header>QWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
